devenv.nix

```diff
@@ -1,8 +1,14 @@
-{ pkgs, nix, lib, config, ... }:
+{ inputs, pkgs, lib, config, ... }:
 
+let
```

Comment:
Why do you think we need shellcheck as part of the repo?

Reply:
Thought it was serving as an example. Removing.